docs/src/components/PropTable.js

```diff
@@ -126,7 +135,22 @@ export default function PropTable({ props: properties, Component }: Props) {
                     <Td shrink border={!description}>
                       <Box>
                         <Text overflow="normal" bold leading="tall">
-                          <code>{name}</code>
+                          {id ? (
+                            <Link
+                              href={`#${id}`}
+                              onClick={({ event }) => {
+                                event.preventDefault();
+                                const elem = document.getElementById(id);
+                                if (elem) {
+                                  elem.scrollIntoView({ behavior: 'smooth' });
```

Comment:
@mminute Could we also add `block: 'start'`? It was confusing to me that the headings didn't appear on the top.